src/Entities/Simulation/SimulationSingleEventType.php

```diff
@@ -0,0 +1,104 @@
+ * | Altering this code will result in changes being overwritten |
+ * |-------------------------------------------------------------|.
+ */
+
+namespace Paddle\SDK\Entities\Simulation;
+
+use Paddle\SDK\PaddleEnum;
+
+/**
+ * @method static SimulationSingleEventType AddressCreated()
+ * @method static SimulationSingleEventType AddressImported()
+ * @method static SimulationSingleEventType AddressUpdated()
+ * @method static SimulationSingleEventType AdjustmentCreated()
+ * @method static SimulationSingleEventType AdjustmentUpdated()
+ * @method static SimulationSingleEventType BusinessCreated()
+ * @method static SimulationSingleEventType BusinessImported()
+ * @method static SimulationSingleEventType BusinessUpdated()
+ * @method static SimulationSingleEventType CustomerCreated()
+ * @method static SimulationSingleEventType CustomerImported()
+ * @method static SimulationSingleEventType CustomerUpdated()
+ * @method static SimulationSingleEventType DiscountCreated()
+ * @method static SimulationSingleEventType DiscountImported()
+ * @method static SimulationSingleEventType DiscountUpdated()
+ * @method static SimulationSingleEventType PayoutCreated()
+ * @method static SimulationSingleEventType PayoutPaid()
+ * @method static SimulationSingleEventType PriceCreated()
+ * @method static SimulationSingleEventType PriceImported()
+ * @method static SimulationSingleEventType PriceUpdated()
+ * @method static SimulationSingleEventType ProductCreated()
+ * @method static SimulationSingleEventType ProductImported()
+ * @method static SimulationSingleEventType ProductUpdated()
+ * @method static SimulationSingleEventType ReportCreated()
+ * @method static SimulationSingleEventType ReportUpdated()
+ * @method static SimulationSingleEventType SubscriptionActivated()
+ * @method static SimulationSingleEventType SubscriptionCanceled()
+ * @method static SimulationSingleEventType SubscriptionCreated()
+ * @method static SimulationSingleEventType SubscriptionImported()
+ * @method static SimulationSingleEventType SubscriptionPastDue()
+ * @method static SimulationSingleEventType SubscriptionPaused()
+ * @method static SimulationSingleEventType SubscriptionResumed()
+ * @method static SimulationSingleEventType SubscriptionTrialing()
+ * @method static SimulationSingleEventType SubscriptionUpdated()
+ * @method static SimulationSingleEventType TransactionBilled()
+ * @method static SimulationSingleEventType TransactionCanceled()
+ * @method static SimulationSingleEventType TransactionCompleted()
+ * @method static SimulationSingleEventType TransactionCreated()
+ * @method static SimulationSingleEventType TransactionPaid()
+ * @method static SimulationSingleEventType TransactionPastDue()
+ * @method static SimulationSingleEventType TransactionPaymentFailed()
+ * @method static SimulationSingleEventType TransactionReady()
+ * @method static SimulationSingleEventType TransactionUpdated()
+ */
+final class SimulationSingleEventType extends PaddleEnum
```

Comment:
Can we check if this is the same as `EventTypeName`

Reply:
Yes, updated and dropped this class as they should match